### PR TITLE
Pop up

### PR DESCRIPTION
**Pull request by dafen173**
### Pop Up

[Demo](https://dafen173.github.io/pop-up/)
[Code base](https://github.com/dafen173/frontend-2021-homeworks/tree/pop-up/submissions/dafen173/pop-up2)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/dafen173/frogger-game/app.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
+// Enemies our player must avoid.........
```
Comment (lenadgit) [inline submissions/dafen173/frogger-game/app.js:1]: this file belongs to another task, could you please remove it

**Review by lenadgit — COMMENTED** [review]
Almost well done) , simple and stylish design, but some bugs are present and need fixes.

File: submissions/dafen173/frogger-game/app.js (hunk 1 above)
Comment (lenadgit) [reply submissions/dafen173/frogger-game/app.js:1]: Impossible to open and close the popup by keyboard. Could you please fix it?

File: submissions/dafen173/pop-up2/index.html [hunk 2]
```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="uk">
```
Comment (lenadgit) [inline submissions/dafen173/pop-up2/index.html:2]: Is English (United Kingdom) the  language for content  ?   The most common language which used is EN

**Review by lenadgit — CHANGES_REQUESTED** [review]

File: submissions/dafen173/pop-up2/index.html [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+	<head>
+		<title>PopUp menu</title>
+		<meta charset="UTF-8">
+		<link rel="stylesheet" href="style.css" />
```
Comment (lenadgit) [inline submissions/dafen173/pop-up2/index.html:6]: The single tags should be without slashes, according to HTML5 specification. Could you please remove all slashes where they were added?

**Review by lenadgit — CHANGES_REQUESTED** [review]

File: submissions/dafen173/pop-up2/style.css [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+}
+
+.MorePopUp {
+	display: block;
+	margin-top: 10px;
+	line-height: 35px;
+	text-align: center;
+	width: 280px;
+	background-color: #eee;
+	font-size: 16px;
```
Comment (lenadgit) [inline submissions/dafen173/pop-up2/style.css:153]: could you please add `cursor: pointer;`

**Review by lenadgit — CHANGES_REQUESTED** [review]

File: submissions/dafen173/pop-up2/style.css [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+.hiddenBlock,
+.line {
+	display: none;
+}
+
+.line {
+	color: #eee;
+	width: 200px;
+	opacity: 0.5;
+
```
Comment (lenadgit) [inline submissions/dafen173/pop-up2/style.css:165]: could you please remove an empty line

**Review by lenadgit — CHANGES_REQUESTED** [review]

**Comment by OleksiyRudenko** [comment]
@dafen173 this PR contains commits from 1+ tasks. We need to fix this.
Please feel free revisiting submission instructions and seek for help in STudents' chat

File: submissions/dafen173/pop-up2/style.css (hunk 5 above)
Comment (dafen173) [reply submissions/dafen173/pop-up2/style.css:165]: Thanks for the review!
I will correct

**Comment by dafen173** [comment]
> @dafen173 this PR contains commits from 1+ tasks. We need to fix this.
> Please feel free revisiting submission instructions and seek for help in STudents' chat

fixed it
it's okay now?

File: submissions/dafen173/frogger-game/app.js (hunk 1 above)
Comment (dafen173) [reply submissions/dafen173/frogger-game/app.js]: > Impossible to open and close the popup by keyboard. Could you please fix it?

seems to have corrected all the points you indicated
it's okay now?

**Comment by dafen173** [comment]
to lenadgit

seems to have corrected all the points you indicated
it's okay now?

File: submissions/dafen173/pop-up2/style.css (hunk 5 above)
Comment (dafen173) [reply submissions/dafen173/pop-up2/style.css]: > could you please remove an empty line

seems to have corrected all the points you indicated
it's okay now?

File: submissions/dafen173/pop-up2/index.html [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+
+				<div class="triangle">
+					<div class="empty"></div>
+				</div>
+
+
+			</div>
+		</nav>
+	</body>
+</html>
```
Comment (lenadgit) [inline submissions/dafen173/pop-up2/index.html:154]: If you see this symbol 
empty line   that means the empty line needs at the end of the file. This line is necessary for webpack (gulp, etc). I will add this symbols in these case and merge

File: submissions/dafen173/pop-up2/style.css [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+	height: auto;
+	margin-right: auto;
+	margin-left: auto;
+}
+
+.captions {
+	margin-top: 5px;
+	text-align: center;
+}
+
```
Comment (lenadgit) [inline submissions/dafen173/pop-up2/style.css:124]: a good practice - do not leave so big indentations between selectors

File: submissions/dafen173/pop-up2/index.html [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	</head>
+	<body>
+		<nav>
+			<div class="menu">
+				<div class="menuItem" tabindex="1">Mail</div>
+				<div class="menuItem" tabindex="2">Photos</div>
+				
+
+				<input type="checkbox" id="ButtonPopUp" class="popUp" tabindex="3"/>
```
Comment (lenadgit) [inline submissions/dafen173/pop-up2/index.html:16]: all single tags should be without slashes - according to HTML5 specification. Slashes for single tags uses only for HTML4 or lower versions.
Could you please fix it?

**Review by lenadgit — CHANGES_REQUESTED** [review]

**Review by lenadgit — CHANGES_REQUESTED** [review]
Would be better to use prettier for formatting your files

**Comment by OleksiyRudenko** [comment]
@dafen173 Please let us know until 20:00 today if you are going to submit any changes into this PR before end of the day February 27. This would help us to plan mentors' limited capacity accordingly.

**Comment by OleksiyRudenko** [comment]
Feel free to reopen this PR once you feel ready to submit any updates.

**Pull request closed without merging**